stable/polaris/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: 5.1.0
-appVersion: "5.1"
+version: 5.2.0
```

Comment:
we do not have breaking changes jumping from `5.1` to `7.0` right?

Reply:
Correct